Background Scripts/listUsersHavingSpecificRole/script.js

```diff
@@ -0,0 +1,11 @@
+function getUsersBasedOnrole(role){
+	var itilUsers = [];
+	var userGR = new GlideRecord('sys_user');
+	userGR.addEncodedQuery('roles.name=itil');
```

Comment:
Have you tested this query? I'm not sure it will work since roles is a list field

Reply:
Yes Wiz,
I have tested this function is working completely fine and returns the list of users having itil role